pkgs/development/python-modules/langgraph-checkpoint-sqlite/default.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  langgraph-checkpoint,
+  aiosqlite,
+  pytest-asyncio,
+  pytestCheckHook,
+  langgraph-sdk,
+  poetry-core,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "langgraph-checkpoint-sqlite";
+  version = "1.0.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.10";
+
+  src = fetchFromGitHub {
+    owner = "langchain-ai";
+    repo = "langgraph";
+    rev = "refs/tags/checkpointsqlite==${version}";
+    hash = "sha256-TUiZOf34jhs+nkeTsprtTdoVqDt7kZd8NxYLKX4l0kQ=";
+  };
+
+  sourceRoot = "${src.name}/libs/checkpoint-sqlite";
+
+  build-system = [ poetry-core ];
+
+  dependencies = [ langgraph-checkpoint ];
+
+  pythonImportsCheck = [ "langgraph.checkpoint.sqlite" ];
+
+  nativeCheckInputs = [
+    aiosqlite
```

Comment:
`aiosqlite` should go to `dependencies`.
https://github.com/langchain-ai/langgraph/blob/d9adebed846e4e822106fbde7ce33e444dcecf06/libs/checkpoint-sqlite/pyproject.toml#L11-L14

Reply:
Fixed.